src/components/Bulk/Preview/__tests__/Preview.test.jsx

```
@@ -1,5 +1,6 @@
 import { BrowserRouter } from 'react-router-dom';
 import { screen, fireEvent } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
```

ESLint

[error] Unable to resolve path to module '`@testing-library/user-event`'.

(import-x/no-unresolved)